public/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<title>about me</title>
+
+<meta name="Sally Moore - about me" content="Web developer with a Ph.D. in clinical psychology. Likes puppies." charset="UTF-8">
+</head>
+
+<h1>Sally A. Moore, Ph.D.</h1>
+
+<header>
+  <a href="/">home</a>&nbsp;&nbsp;&nbsp;
```

Comment:
Semantically, I'd like to see this list of links enclosed by a tag describing their relationship, like `<nav>`